code/modules/clothing/head/hardhat.dm

```diff
@@ -90,9 +90,12 @@
 /obj/item/clothing/head/hardhat/atmos
 	name = "atmospheric technician's firefighting helmet"
 	desc = "A firefighter's helmet, able to keep the user cool in any situation."
+	icon = 'icons/obj/clothing/head/utility.dmi'
 	icon_state = "hardhat0_atmos"
 	item_state = "hardhat0_atmos"
```

Comment:
Unneeded line of code, the other "item_state" variable is correct

Reply:
The highlighted line of code and the comment are not related? But I'm assuming you meant `item_state` to be deleted, if so then fair enough.